Content: New Hai advanced start (#7721)
endless-sky/endless-sky: data/hai/hai missions.txt

			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
			choice
				`	"Yes, I am."`
					to display
						not "hai start"
					goto outsider
				`	"Not really, I grew up here."`
					to display
						has "hai start"
					goto clothe
				`	"What do you mean 'the outside?'"`
			
			`	"Outside of Hai space. You don't look quite like most humans who live here, unless there's some new fashion trend going on that I don't know about."`
			choice
				`	"Oh, yes. I do come from outside."`
					to display
						not "hai start"
				`	"Not really, I grew up here, and made these clothes for myself."`
					to display
						has "hai start"
					goto outsider
			
			label clothe
			`	"That's weird, your clothes don't look like most do here." He says.`
				to display
					has "hai start"
			label outsider

> 			`	You explain to him that you grew up making clothes, meaning you grew tired of the usual clothing you kept making and tried to make some more original ones, including the one you're wearing right now. You add that whilst you may not have been born in human space, your family made sure you learned of the origins of your specie, meaning you know human space like your pocket.`

Author: both?

> 			`	You explain to him that you grew up making clothes, meaning you grew tired of the usual clothing you kept making and tried to make some more original ones, including the one you're wearing right now. You add that whilst you may not have been born in human space, your family made sure you learned of the origins of your specie, meaning you know human space like your pocket.`

Reviewer: I've just never heard it and wasn't sure.

If it's a saying in its own right, it works. Just seemed especially odd if it wasn't lol

> 			`	You explain to him that you grew up making clothes, meaning you eventually grew tired of the usual clothing you kept making. Eventually you tried to make some more original ones, including the one you're wearing right now. You add that whilst you may not have been born in human space, your family made sure you learned about the origins of your species; you know human space like your pocket.`

Reviewer 2: I don't like the repeated `making` and `eventually`.

Author: ```suggestion
			`	You explain to him that you grew up making clothes, meaning you eventually grew tired of the usual clothing you kept making. Once you gained enough experience you started creating more original ones, including the one you're wearing right now. You add that whilst you may not have been born in human space, your family made sure you learned about the origins of your species; you know human space like your pocket.`
```

Reviewer 2: Much better.

mission "Hai Waypoint Warning"
	landing
	invisible
	to offer
		has "hai space start"
	to complete
		never
	on enter "Waypoint"
		fail
		conversation
			`As the <ship> drops out of hyperspace, your dashboard is filled with an array of warning lights. It isn't hard to find the cause; even from this distance, the light of the wormhole is as clear as crystal, tinting everything around it a pale purple. The arms of the vortex rotate seemingly lethargically, but you know that, close up, the bands of energy that make it up must be moving several orders of magnitude faster than the top speed of your ship.`

> 			`	Your sightseeing is interupted by a hail on an encrypted frequency. You open it and are met with what you vaguely recognize as a short greeting in the Hai language.`

Reviewer 3: ```suggestion
			`	Your sightseeing is interrupted by a hail on an encrypted frequency. You open it and are met with what you vaguely recognize as a short greeting in the Hai language.`
```

			choice
				`	"Could you repeat that?"`
				`	"Sorry, I don't speak Hai."`
			`	You receive a reply in the form of a hearty chuckle. "My bad, friend," the Hai hailing you says. "Used to be that anyone who grew up in Hai space knew Hai, but I guess that isn't really true anymore, what with Greenwater and such.`
			`	"Anyway, back on topic. You've probably already heard this a hundred times, so I'll be brief: As long as you're in human space, the Hai don't exist, and any outfits and ships you have are experimental prototypes. Got it?"`
			choice
				`	"Got it."`
					goto easy
				`	"What's wrong with telling the truth?"`
			`	The Hai lets out another laugh, this time more nervous. "You know, there's some people out there who'd be able to say what you just did with a straight face."`
			choice
				`	"Alright, I'll be on my way."`
					goto easy
				`	"I'm serious."`

> 			`	While you can't see your conversation partner over the comlink, you get the feeling that they'd be pinching the bridge of their nose right now. "You know, just a few decades ago, you'd be hard-pressed to find any captain in Hai space who'd even consider revealing the Hai's presence as anything but a disaster. Nowadays, it feels like even Hai are clamoring for the chance to go public."`

Reviewer 4: ```suggestion
			`	While you can't see your conversation partner over the commlink, you get the feeling that they'd be pinching the bridge of their nose right now. "You know, just a few decades ago, you'd be hard-pressed to find any captain in Hai space who'd even consider revealing the Hai's presence as anything but a disaster. Nowadays, it feels like even Hai are clamoring for the chance to go public."`
```

			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
			choice
				`	"Yes, I am."`
					to display
						not "hai space start"
					goto outsider
				`	"Not really. I grew up here."`
					to display
						has "hai space start"
					goto clothe
				`	"What do you mean 'the outside?'"`
			
			`	"Outside of Hai space. You don't look quite like most humans who live here, unless there's some new fashion trend going on that I don't know about."`
			choice
				`	"Oh, yes. I do come from outside."`
					to display
						not "hai space start"
				`	"Not really. I grew up here."`
					to display
						has "hai space start"
					goto outsider
			
			label clothe
			`	"That's weird. Your clothes don't look like most do here," he says.`
			label outsider

> 			`	You explain to him that these clothes used to belong to your father, who was a merchant from human space; your mother kept some for you when he left. You add that while you may not have been born in human space, your family made sure you learned about the origins of your species; you know human space like the back of your hand.`

Reviewer 2: But you still need to buy all the local maps to know where the systems are?

Reviewer 5: Yeah, I don't think you should assume that the player has this knowledge, it's a bit of a fourth wall break.

Author: I did this bc I assume many missions rely on some sort of knowledge of human space
```suggestion
			`	You explain to him that these clothes used to belong to your father, who was a merchant from human space; your mother kept some for you when he left. You add that while you may not have been born in human space, your family made sure you learned about the origins of your species; you would be able to find your way in human space.`
```

Reviewer 5: To be honest I don't think the second sentence is really necessary at all. 
```suggestion
			`	You explain to him that these clothes used to belong to your father, who was a merchant from human space; your mother kept some for you when he left.`
```

				to display
					has "hai space start"

> 			`	"Good. My name is Elliot." You introduce yourself to Elliot and shake hands.`

Reviewer 5: Not sure if this would be better as a branched conversation.
```suggestion
			`	"Good. My name is Elliot." You introduce yourself to Elliot and shake hands.`
				to display
					has "human space start"
			`	He sighs. "I was hoping to get someone from human space. Well, a captain's a captain, I guess. My name is Elliot." You introduce yourself to Elliot and shake hands.
				to display
					has "hai space start"
```

Author: shouldnt he be excited to find a brother from hai space?

Reviewer 5: Elliot isn't from Hai space, iirc. He's originally from human space, but ran away as a teenager.

> 			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
> 			choice
> 				`	"Yes, I am."`
> 					to display
> 						not "hai space start"
> 					goto outsider
> 				`	"Not really. I grew up here."`
> 					to display
> 						has "hai space start"
> 					goto clothe
> 				`	"What do you mean 'the outside?'"`
> 			
> 			`	"Outside of Hai space. You don't look quite like most humans who live here, unless there's some new fashion trend going on that I don't know about."`
> 			choice
> 				`	"Oh, yes. I do come from outside."`
> 					to display
> 						not "hai space start"
> 				`	"Not really. I grew up here."`
> 					to display
> 						has "hai space start"
> 					goto outsider
> 			
> 			label clothe
> 			`	"That's strange. You don't look like most do here," he says.`
> 			label outsider
> 			`	You explain to him that these clothes used to belong to your father, who was a merchant from human space; your mother kept some for you when he left.`
> 				to display
> 					has "hai space start"
> 			`	"Good. My name is Elliot." You introduce yourself to Elliot and shake hands.`
> 				to display
> 					has "human space start"
> 			`	He sighs. "I was hoping to get someone from human space. Well, a captain's a captain, I guess. My name is Elliot." You introduce yourself to Elliot and shake hands.`
> 				to display
> 					not "human space start"

Reviewer 3: This is a clumsy way of handling this mission, and it makes some pretty big assumptions about the player. (Have I really not bought any clothes the entire time I've been a captain?) The clothing thing isn't that vital to the mission, either; Arthur and Kiru don't approach you just because you're a captain from the outside.
```suggestion
			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
			choice
				`	"Yes, I am."`
					to display
						not "hai space start"
					goto outsider
				`	"Not really. I grew up here."`
					to display
						has "hai space start"
					goto clothe
				`	"What do you mean 'the outside?'"`
			
			`	"Outside of Hai space. You don't look quite like most humans who live here, unless there's some new fashion trend going on that I don't know about."`
			choice
				`	"Oh, yes. I do come from outside."`
					to display
						not "hai space start"
				`	"Not really. I grew up here."`
					to display
						has "hai space start"
					goto outsider
			
			label clothe
			`	"That's strange. You don't look like most do here," he says.`
			label outsider
			`	You explain to him that these clothes used to belong to your father, who was a merchant from human space; your mother kept some for you when he left.`
				to display
					has "hai space start"
			`	"Good. My name is Elliot." You introduce yourself to Elliot and shake hands.`
				to display
					has "human space start"
			`	He sighs. "I was hoping to get someone from human space. Well, a captain's a captain, I guess. My name is Elliot." You introduce yourself to Elliot and shake hands.`
				to display
					not "human space start"
```

> 					not "human space start"

Author: yeah I got the labels confused but I don't agree with branches being clearer, I think it could be about being used to them instead of the to display? I got a bit confused when trying to make it branch like personally

			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
			choice
				`	"Yes, I am."`
					to display
						not "hai start"
					goto outsider

> 				`	"Not really, I grew up here."`

Reviewer 6: ```suggestion
				`	"Not really. I grew up here."`
```

					to display
						has "hai start"
					goto clothe
				`	"What do you mean 'the outside?'"`
			
			`	"Outside of Hai space. You don't look quite like most humans who live here, unless there's some new fashion trend going on that I don't know about."`
			choice
				`	"Oh, yes. I do come from outside."`
					to display
						not "hai start"
				`	"Not really, I grew up here and made these clothes for myself."`
					to display
						has "hai start"
					goto outsider
			
			label clothe

> 			`	"That's weird, your clothes don't look like most do here." He says.`

Reviewer 6: ```suggestion
			`	"That's weird. Your clothes don't look like most do here," he says.`
```

				to display
					has "hai space start"
			`	"Good. My name is Elliot." You introduce yourself to Elliot and shake hands.`
				to display
					has "human space start"
			`	He sighs. "I was hoping to get someone from human space. Well, a captain's a captain, I guess. My name is Elliot." You introduce yourself to Elliot and shake hands.
				to display

> 					has "hai space start"

Reviewer 2: Shouldn't this be `not "human space start"` or something? In case a plugin has some other start neither option would display.

Author: ```suggestion
					not "human space start"
```
I agree, but this is probably based on him recognizing your hai style clothing? idk

Reviewer 5: Good catch, thanks.

	destination "Allhome"
	to offer
		has "event: hai-human resolution announced"
		or
			not "hai start"
			"combat rating" > 500
	on offer
		conversation
			`Walking through the <origin> spaceport, you notice that almost half of the people you see are humans, unlike other Hai worlds where the Hai outnumber humans by a sizable degree. A human dressed in a business suit shouts your name from across the street and approaches you.`
			branch "hai start"
				has "hai start"
			`	"Captain <last>, fantastic to see you again! I noticed you right away when I laid my eyes upon you and I must say, I'd like to thank you again for what you've done."`
			`	The man is David Turner, who was involved in the movie production during the Hai crisis. Almost reflexively he goes to grab a business card from his pocket but then seems to remember and puts it back away. Seems to be exactly the same card as last time though.`
			label "hai start"

> 			`	"Hello, captain. I come to you with a business opportunity you would do well to listen to." He grabs his business card to show it to you, and presents himself. "My name is David Turner, you may know me for my involvement with the Hai cover movie? I come to you because you seem to have quite a bit of combat experience, which would be of great help for this business venture." He then adds something, in a low tone, "Also, you are not a pirate saying that they redeemed themselves..."`

Reviewer: ```suggestion
			`	"Hello, captain. I come to you with a business opportunity you would do well to listen to." He grabs his business card to show it to you and presents himself. "My name is David Turner, you may know me for my involvement with the Hai cover movie? I come to you because you seem to have quite a bit of combat experience, which would be of great help for this business venture." He then adds, in a low tone, "Also, you are not a pirate saying that they redeemed themselves...`
```
